pkg/application-manager/handlers.go

```diff
@@ -223,6 +221,7 @@ func (h *Handlers) updateApp(params application.UpdateAppParams, principal inter
 	}
 	e.Status = entitystore.StatusREADY
 	updatedEntity := applicationModelOntoEntity(params.Body)
+	updatedEntity.OrganizationID = params.XDISPATCHORGID
```

Comment:
Do we really want to update the organization id? I imagine that for security reasons we wouldn't want one object to become a part of a different organization. Users should recreate the resource in the new org.

Reply:
It looks misleading but we don't allow an entity to be updated with a different orgID. We query if the entity exists within the org specified by the user and then update it. I could do `updatedEntity.OrganizationID = e.OrganizationID` to be less ambiguous.

Reply:
Perhaps even less ambiguous would be if we added the orgID to the model, so that we don't need to tack it on at all.